example/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "example",
+  "version": "0.0.0",
+  "description": "",
+  "main": "build.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "requirejs": "^2.1.13"
```

Comment:
This and other dev dependencies will be added by me in upcoming, fresh 2017 PR, but in root `package.json`.